templates/config/traefik.toml.erb

```diff
@@ -1,27 +1,3 @@
-logLevel = "INFO"
-defaultEntryPoints = [<%= '"https",' if @config['ssl']['enabled'] %>"http"]
-
-[api]
-
-[entryPoints]
-  [entryPoints.http]
-  address = ":80"
-<% if @config['ssl']['enabled'] -%>
-    [entryPoints.http.redirect]
-    entryPoint = "https"
-  [entryPoints.https]
-  address = ":443"
-  [entryPoints.https.tls]
-<% end -%>
-
-[retry]
-
-[docker]
-endpoint = "unix:///var/run/docker.sock"
-domain = "local"
-watch = true
-exposedbydefault = false
-
 <% if @config['ssl']['enabled'] -%>
```

Comment:
[certificatesResolvers.myresolver.acme]
  email = "your-email@your-domain.org"
  storage = "acme.json"
  [certificatesResolvers.myresolver.acme.httpChallenge]
    # used during the challenge
    entryPoint = "web"